### DISCUSSION TITLE
Suspension of my GitHub account

### DISCUSSION THREAD

**Discussion by Rjshri** [community/community#135367] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub Team,

I am writing to express my deep concern and frustration regarding the suspension of my GitHub account (username: Shivyoddha). Despite several weeks of waiting, I have yet to receive any response or update from your support team.

My account suspension has severely impacted my work, bringing my projects to a halt for an extended period. What is most concerning is the lack of communication and transparency regarding the issue or the expected timeline for resolution. As a long-time user who relies heavily on your platform, I find this lack of support unacceptable.

I fully understand that this community page may not be the typical forum for resolving such issues, but after experiencing significant delays and one-sided communication from the support team, I feel compelled to voice my concerns here. The current situation leaves me with no clarity on when, or even if, my issue will be addressed.

I urge you to take immediate action to resolve this matter. At the very least, please provide me with a clear explanation of the suspension and an estimated timeline for resolution. My work depends on access to my account, and the ongoing delay is causing significant disruption.

Thank you for your attention to this matter. I look forward to your prompt response.

PS: This is my friend's account.

**Comment by Rjshri** [comment]
My ticket id is: 2894258

**Reply by SAHELISAHAA** [reply]
@Rjshri same issue bro , if it is more than a week their is less change that they will response

**Reply by ghost** [reply]
I am in the same boat. I have been waiting for almost three weeks at this point! Someone else here was waiting for 88 days.

There is no transparency, no indication of where your ticket is in the queue. It is genuinely unbelievable.

**Reply by ghost** [reply]
That is not a bad idea. A tweet is probably better than LinkedIn.

**Reply by SAHELISAHAA** [reply]
@owenmcgrath2004 i raised a discussion https://github.com/orgs/community/discussions/135371

**Reply by still-hash-ir** [reply]
I am facing the same issue. I create a new account (this) to start a discussion here: https://github.com/orgs/community/discussions/134510

Also tweeted about this but no response yet: https://x.com/_hash_ir_/status/1821209794960564227

It has been more than 2 weeks for me and this is seriously hurting my work!
